packages/angular_devkit/build_angular/src/builders/server/schema.json

```diff
@@ -189,6 +189,11 @@
       "description": "Extract all licenses in a separate file, in the case of production builds only.",
       "default": true
     },
+    "buildOptimizer": {
+      "type": "boolean",
+      "description": "Enables advanced build optimizations.",
+      "default": false
```

Comment:
For consistence should we set this to `true`? 

//cc @clydin

Reply:
i think so.